### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by mattheu**
Just some styling for the built-in WP comments. We can add some fancy inline comment stuff later.

Ref: https://github.com/humanmade/hm-handbook-theme/issues/5

### CODE REVIEW

File: assets/src/styles/components/_comments.scss [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	@extend .Btn;
+	@extend .Btn-Small;
+	float: right;
+}
+
+.comment-awaiting-moderation {
+	background: $hm-light-grey;
+	border-radius: 2px;
+	padding: $base-line-height / 2 $gutter-width;
+	margin-left: calc( -3rem - #{ $gutter-width } );
```
Comment (sambulance) [inline assets/src/styles/components/_comments.scss]: Assuming the 3rem here is from the avatar width? Might be worth making the avatar size a variable. 
1. Easier to read and understand when used away from other avatar declarations (like this)
2. If we decide to change the size, can be done in one place
3. Can also be overridden by a child theme

File: comments.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+				'short_ping'  => true,
+				'avatar_size' => 54,
+			) );
+			?>
+		</ol><!-- .comment-list -->
+
+	<?php endif; // have_comments() ?>
+
+	<?php if ( ! comments_open() && get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) : ?>
+		<p class="no-comments"><?php _e( 'Comments are closed.', 'twentyfifteen' ); ?></p>
```
Comment (sambulance) [inline comments.php]: Update text domain

Comment (mattheu) [inline comments.php]: oops!

File: assets/src/styles/components/_comments.scss (hunk 1 above)
Comment (mattheu) [inline assets/src/styles/components/_comments.scss]: 👍

File: assets/src/styles/theme.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-$images-path: "./../../../vendor/hm-pattern-library/assets/images" !default;
+$hm-images: "./../../../vendor/hm-pattern-library/assets/images";
```
Comment (sambulance) [inline assets/src/styles/theme.scss]: Was this supposed to change? `$images-path` is still being used in stylesheets

**Comment by sambulance** [comment]
Sorry I've only just seen this. Added a few comments. Master will need merging too.

File: assets/src/styles/theme.scss (hunk 3 above)
Comment (mattheu) [inline assets/src/styles/theme.scss]: Nope. Must have been a merge fail

**Comment by mattheu** [comment]
@sambulance No worries! Nothing urgent :) Thanks for your review!

I've fixed up the issues you flagged and synced with master. Merging!

**Pull request merged**